src/components/Rocket.js

```diff
@@ -0,0 +1,34 @@
+import { useDispatch } from 'react-redux';
+import { createReserve } from '../redux/rockets/rocketReducers';
+
+const Rock = (props) => {
+  const rocket = props;
+  const dispatch = useDispatch();
+  const { reserve } = rocket.rock;
+  const handleReserve = (e) => {
+    e.target.classList.toggle('reservedButton');
+    const toggle = !reserve;
+    const setReserve = (() => ({ ...rocket.rock, reserve: toggle }));
+    dispatch(createReserve(setReserve()));
+  };
```

Comment:
Actions should only be dispatched once as mentioned above, please fix this.

<img width="311" alt="Screen Shot 2022-08-26 at 8 18 59" src="https://user-images.githubusercontent.com/82426682/186912651-d7d74ee3-21a0-46c6-9532-7a4b7b100650.png">

<img width="218" alt="Screen Shot 2022-08-26 at 8 19 15" src="https://user-images.githubusercontent.com/82426682/186912711-b0df0ae1-00e6-4362-831b-0254a027ff08.png">